WebDriverAgentLib/Categories/XCUIElement+FBScrolling.m

```diff
@@ -90,8 +97,14 @@ - (BOOL)fb_scrollToVisibleWithNormalizedScrollDistance:(CGFloat)normalizedScroll
 
   XCElementSnapshot *targetCellSnapshot = self.fb_parentCellSnapshot;
   NSArray<XCElementSnapshot *> *cellSnapshots = [scrollView fb_descendantsMatchingType:XCUIElementTypeCell];
+
+  if (cellSnapshots.count == 0) {
+    // For the home screen, cells are actually of type XCUIElementTypeIcon
+    cellSnapshots = [scrollView fb_descendantsMatchingType:XCUIElementTypeIcon];
```

Comment:
True. However, all descendants is kind of a measure of last resort, because it will also find any subelements that aren't actually "cells." Before we do that, I want to try finding icons, which are essentially cells anyways. If this isn't necessary, then one could argue `[scrollView fb_descendantsMatchingType:XCUIElementTypeCell]` isn't necessary either.